command/volume_delete.go

```diff
@@ -70,6 +73,8 @@ func (c *VolumeDeleteCommand) Name() string { return "volume delete" }
 func (c *VolumeDeleteCommand) Run(args []string) int {
 	flags := c.Meta.FlagSet(c.Name(), FlagSetClient)
 	flags.Usage = func() { c.Ui.Output(c.Help()) }
+	var secrets string
```

Comment:
We usually store flags in the command `struct`: https://github.com/hashicorp/nomad/blob/v1.1.6/command/job_status.go#L23-L26